news/2017-10-13-osu-world-cup-2017-registrations-now-open.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "osu! World Cup 2017: Registrations now open!"
+date: 2017-10-13 13:00:00 +0000
+---
+It is time again for this year's official osu! World Cup. With a brand new design, osu!'s fiercest and most exciting competition is just getting started
+
+![](/wiki/shared/news/banners/owc_2017_banner.png)
+
+For more details, refer to the **[osu! World Cup 2017 wiki article](/wiki/Tournaments/OWC/2017/)**.
+You can discuss this event in the **[official forum thread](https://osu.ppy.sh/community/forums/topics/653192)**.
+
+## Tournament Schedule
+
++ Registration Phase: 13. - 29. October 2017
++ Live Drawings: 12. November 2017 14:00 UTC+0
++ Group Stage: 18. - 19. November 2017
++ Round of 16: 25. - 26. August 2017
++ Quarterfinals: 2. - 3. December 2017
++ Semifinals: 9. - 10. December 2017
++ Finals: 15. - 17. December 2017
+
+## Prizes
+
++ **1st:** $300 per team member, exclusive osu! tumbler and pins, profile badge, "osu! Champion" user title for one year
++ **2nd:** $160 per team member, exclusive osu! tumbler and pins, profile badge
++ **3rd:** $80 per team member, exclusive osu! tumbler and pins, profile badge
+
+## How do I register?
+
++ **[Click here to sign-up!](http://osu.ppy.sh/community/tournaments/12)**
++ Do not create teams in advance! Teams will be formed after registration phase concluded by country’s team captains.
++ Hand in your registration before **29. October 2017.**
+
+Note that you can only participate if you are of a **osu! global ranking of #5000 or higher and did not violate the [osu! community rules](/wiki/Rules) in the past 12 months**.
+
+If your registration successfully passed our evaluation, you will be put on the candidate list of your country. Whenever a captain for your country has been assigned, the list will be sent to them. This may take a while, so please be patient.
```

Comment:
`candidate list of your country.` -> `candidate list for your country.`
... 🤷‍♂️ it might just be me that thinks the wording is weird